include/PikaRevision.h

```diff
@@ -0,0 +1,8 @@
+/* THIS FILE IS AUTOGENERATED - DO NOT EDIT */
```

Comment:
What am I supposed to do with this file? Delete it? I don't remember this ever showing up before...